spec/rails_helper.rb

```diff
@@ -140,6 +140,11 @@
     clear_enqueued_jobs
   end
 
+  unless ENV['CI']
```

Comment:
Is there a way to easily override this locally so we can run all of the specs as required?

Reply:
Yes, will add instructions to the readme.